### PR TITLE
AWS release 9.0.0 with Kubernetes 1.15.5

### PR DESCRIPTION
**Pull request by tfussell**
Towards https://github.com/giantswarm/giantswarm/issues/6431

### CODE REVIEW

**Comment by tfussell** [comment]
Should 8.X remain active? What about 7.X? Not sure about the deprecation process.

**Comment by pipo02mix** [comment]
We only maintain the last two majors, so `7.X` goes deprecated

File: aws.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-- active: false
+- active: true
```
Comment (kopiczko) [inline aws.yaml]: aws-operator@5.5.0 is a legacy branch so we'd need to release a flat operator. Please ping @giantswarm/team-magic  when doing so for a while.

@giantswarm/team-magic needs to fix one bug with finalizers before we can deploy that to production. It was discovered yesterday evening. I'll link to the issue once it's created.

Comment (kopiczko) [reply aws.yaml]: Here's the issue https://github.com/giantswarm/giantswarm/issues/7376

Comment (kopiczko) [reply aws.yaml]: Fix is merged and vendored now..

**Comment by MarcelMue** [comment]
I created a separate PR for KVM to make our lives a bit easier.

**Pull request merged**